### ISSUE TITLE
[Bug] Error in release mode

### ISSUE THREAD

**Issue by DavideBicego** [mono0926/barcode_scan2#9]
**Describe the bug**
When launching the scanner in an app built in release mode the scanner is not shown. Look below for the logs.
There seems to be some problem with 
```
Field aspectTolerance_ for de.mintware.barcode_scan.d not found. Known fields are [private double de.mintware.barcode_scan.d.f, private boolean de.mintware.barcode_scan.d.g, private static final de.mintware.barcode_scan.d de.mintware.barcode_scan.d.h, private static volatile e.a.b.a1 de.mintware.barcode_scan.d.i]
```
**To Reproduce**
1. in example/ exec flutter run --release
2. click on the scan button in the action bar

**Expected behavior**
The scanner should be shown

**Logs**
```
example % flutter run --release
Launching lib/main.dart on JSN L21 in release mode...
Running Gradle task 'assembleRelease'...                                
Running Gradle task 'assembleRelease'... Done                      64,2s
✓ Built build/app/outputs/flutter-apk/app-release.apk (7.7MB).
Installing build/app/outputs/flutter-apk/app.apk...              2.465ms

Flutter run key commands.
h Repeat this help message.
c Clear the screen
q Quit (terminate the application on the device).
E/flutter (18978): [ERROR:flutter/lib/ui/ui_dart_state.cc(186)] Unhandled Exception: PlatformException(scan, null, java.lang.reflect.InvocationTargetException
E/flutter (18978): 	at java.lang.reflect.Method.invoke(Native Method)
E/flutter (18978): 	at de.mintware.barcode_scan.ChannelHandler.c(Unknown Source:50)
E/flutter (18978): 	at f.a.c.a.j$a.a(Unknown Source:17)
E/flutter (18978): 	at io.flutter.embedding.engine.e.b.d(Unknown Source:57)
E/flutter (18978): 	at io.flutter.embedding.engine.FlutterJNI.handlePlatformMessage(Unknown Source:4)
E/flutter (18978): 	at android.os.MessageQueue.nativePollOnce(Native Method)
E/flutter (18978): 	at android.os.MessageQueue.next(MessageQueue.java:363)
E/flutter (18978): 	at android.os.Looper.loop(Looper.java:173)
E/flutter (18978): 	at android.app.ActivityThread.main(ActivityThread.java:8178)
E/flutter (18978): 	at java.lang.reflect.Method.invoke(Native Method)
E/flutter (18978): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:513)
E/flutter (18978): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1101)
E/flutter (18978): Caused by: java.lang.RuntimeException: Field aspectTolerance_ for de.mintware.barcode_scan.d not found. Known fields are [private double de.mintware.barcode_scan.d.f, private boolean de.mintware.barcode_scan.d.g, private static final de.mintware.barcode_scan.d de.mintware.barcode_scan.d.h, private static volatile e.a.b.a1 de.mintware.barcode_scan.d.i]
E/flutter (18978): 	at e.a.b.v0.n0(Unknown Source:72)
E/flutter (18978): 	at e.a.b.v0.T(Unknown Source:698)
E/flutter (18978): 	at e.a.b.v0.R(Unknown Source:12)
E/flutter (18978): 	at e.a.b.k0.e(Unknown Source:60)
E/flutter (18978): 	at e.a.b.k0.a(Unknown Source:49)
E/flutter (18978): 	at e.a.b.d1.d(Unknown Source:17)
E/flutter (18978): 	at e.a.b.d1.e(Unknown Source:4)
E/flutter (18978): 	at e.a.b.z.C(Unknown Source:4)
E/flutter (18978): 	at e.a.b.z$a.p(Unknown Source:9)
E/flutter (18978): 	at e.a.b.z$a.o(Unknown Source:0)
E/flutter (18978): 	at de.mintware.barcode_scan.f$b.z(Unknown Source:7)
E/flutter (18978): 	at de.mintware.barcode_scan.ChannelHandler.scan(Unknown Source:69)
E/flutter (18978): 	... 12 more
E/flutter (18978): , null)
E/flutter (18978): #0      StandardMethodCodec.decodeEnvelope (package:flutter/src/services/message_codecs.dart:581)
E/flutter (18978): #1      MethodChannel._invokeMethod (package:flutter/src/services/platform_channel.dart:158)
E/flutter (18978): <asynchronous suspension>
E/flutter (18978): #2      BarcodeScanner._doScan (package:barcode_scan2/platform_wrapper.dart:74)
E/flutter (18978): <asynchronous suspension>
E/flutter (18978): #3      BarcodeScanner.scan.<anonymous closure> (package:barcode_scan2/platform_wrapper.dart:46)
E/flutter (18978): <asynchronous suspension>
E/flutter (18978): 

```

**Environment (please complete the following information):**
 - [ ] iOS? Version: ...
 - [x] Android? Version: 10
 - [x] Real Device? Which?: Huawei JSN L21
 - [ ] Emulator/Simulator? Which?: ...
 - Version of `barcode_scan` 
> v4.0.0
 - Output of `flutter --version`: 
```
Flutter 2.0.5 • channel stable • https://github.com/flutter/flutter.git
Framework • revision adc687823a (10 giorni fa) • 2021-04-16 09:40:20 -0700
Engine • revision b09f014e96
Tools • Dart 2.12.3
```

**Comment by DavideBicego** [comment]
I tried to look into this but I can't find the issue!
**This is blocking!**

**Comment by mono0926** [comment]
By reverting https://github.com/mono0926/barcode_scan2/commit/cea32818174b62326d3f798c264f0e78f1a05aae, this seems to fixed, and released as 4.1.2.
Although this revert might cause https://github.com/firebase/firebase-android-sdk/pull/1534 problem again, I priotized to fix this critical issue.

**Comment by mono0926** [comment]
> By reverting cea3281, this seems to fixed, and released as 4.1.2.
Although this revert might cause firebase/firebase-android-sdk#1534 problem again, I priotized to fix this critical issue.

Fixed both problems and released as 4.1.3 now.

1.Re-apply https://github.com/mono0926/barcode_scan2/commit/cea32818174b62326d3f798c264f0e78f1a05aae : https://github.com/mono0926/barcode_scan2/commit/084371182cc221653d603b6aee65bfcb31609e40
2. Fix the original problem properly: https://github.com/mono0926/barcode_scan2/commit/d540aef8eaf693fb2628449d00aa7a422c6e1d65

**Comment by DavideBicego** [comment]
Thanks!
